cpp/tests/rolling/empty_input_test.cpp

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+#include "cudf/utilities/traits.hpp"
```

Comment:
Should we not change this to use `<>` for headers?

Reply:
autofilled by IDE. will fix this.

Reply:
```suggestion
```